Sara/quickfix/update support (#1067)
returntocorp/semgrep-docs: docs/support.md

# Support

We’re here to help!

We at Semgrep, Inc. want you to love using Semgrep. All paying customers have access to a variety of support channels, technical documentation and an active community of Semgrep users. We’re committed to helping our customers make the most out of their Semgrep subscription.

> ## Support Hours

Reviewer: ```suggestion
## Support hours
```

Semgrep Technical support is available 18 hours a day, 5 days a week from 6am - 12am UTC (Monday to Friday), excluding Semgrep-recognized holidays.

## Contacting Support

All Semgrep customers can contact Semgrep Support through the following methods.

### Slack

Customers who have a private slack channel with Semgrep can open a support case directly from Slack. 

### Web

Customers who log into the Semgrep cloud platform portal can open a support case from the [Help section](https://semgrep.dev/orgs/-/support). 

### Email

For non-urgent inquiries, or when you need to provide detailed information about your issue, you can send an email to our Support team at [support@semgrep.com](mailto:support@semgrep.com).

> ## Community Support

Reviewer: ```suggestion
## Community support
```

> Join our Slack community to chat with the Semgrep maintainers and support engineers! All users are welcome to ask for help in the community Slack group. Semgrep OSS users can file a file bug report for bugs they are running into here.

Reviewer: ```suggestion
[Join the Slack community](https://go.semgrep.dev/slack) to chat with the Semgrep maintainers and support engineers! All users are welcome to ask for help in the community Slack group. Semgrep OSS users can file a file bug report for bugs they are running into here.
```

> ## Status Page

Reviewer: ```suggestion
## Status page
```

> Semgrep’s status page also allows users to subscribe to notifications whenever an incident is created, updated, or resolved. Check out our status page to see any current or past service incidents or to review historical uptime.

Reviewer: ```suggestion
Semgrep’s status page also allows users to subscribe to notifications whenever an incident is created, updated, or resolved. Check out the status page to see any current or past service incidents or to review historical uptime.
```
